### PR TITLE
docs: update the architecture doc

### LINKED ISSUE #292: The code to start sandbox or container in architecture doc can not found in runtime.
image

The code to start sandbox or container in architecture doc can not found in runtime.

### PR DESCRIPTION
**Pull request by liucimin**
Update the k8s create sandbox and container code in runtime.
Fixes #292
Signed-off-by: Liu Cimin lcmjkson@gmail.com

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @liucimin - thanks for raising!

Travis is failing:

```
ERROR: Commit 04da1004f07aa4aef1515c5fd022b52ba1636df9: subject too long (max 75, got 149): "docs: update the architecture doc Update the k8s create sandbox and container code in runtime. Fixes #292 Signed-off-by: Liu Cimin lcmjkson@gmail.com"
ERROR: checkcommits failed. See the document below for help on formatting
commits for the project.
https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format
```

It looks like you've subsequently updated the commit but all you need to do now to make Travis happy is add a blank line between the first line of the commit message and the body:

```
docs: update the architecture doc

Update the k8s create sandbox and container code in runtime.

Fixes #292.

Signed-off-by: Liu Cimin lcmjkson@gmail.com
```

**Comment by jodh-intel** [comment]
Once done,

lgtm

File: architecture.md [hunk 1]
```diff
@@ -273,7 +273,7 @@ With traditional containers, `start` launches a container process in its own set
 1. Communicate with `kata-agent` (through the proxy) to start the container workload
  inside the VM. If, for example, the command to execute inside of the container is `top`,
  the `kata-shim`'s `ReadStdOut()` will start returning text output for top, and
-  `WaitProcess()` will continue to block as long as the `top` process runs.
+    `WaitProcess()` will continue to block as long as the `top` process runs.
```
Comment (sboeuf) [inline architecture.md]: Why have you added a space here?

File: architecture.md [hunk 2]
```diff
@@ -602,24 +602,33 @@ respectively create a virtual machine or a container inside a virtual machine as
...
-		process, err = createPod(ociSpec, runtimeConfig, containerID, bundlePath, console, disableOutput)
+	ociSpec, err := oci.GetOCIConfig(status)
+	if err != nil {
+		return nil, err
+	}
+
+	var sandbox vc.VCSandbox
+
+	if containerType.IsSandbox() {
+		s, err := vci.StartSandbox(ctx, sandboxID)
```
Comment (sboeuf) [inline architecture.md]: I feel that you updated with the wrong code here. Did you use `cli/start.go` instead of `cli/create.go`?

Comment (liucimin) [reply architecture.md]: Yes, u are right.Thx for the sugguestion.
I update the change.

**Comment by sboeuf** [comment]
@liucimin thanks! looks good!

**Comment by jodh-intel** [comment]
Merging - @sboeuf's ack is good enough for me! ;)

**Pull request merged**